.github/workflows/create-lint-wf.yml

```diff
@@ -28,6 +28,8 @@ jobs:
           sudo ln -s /tmp/nextflow/nextflow /usr/local/bin/nextflow
 
       - name: Run nf-core/tools
+        env:
+          GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Sync WF uses `${{ secrets.nf_core_bot_auth_token }}`. Does this token not have sufficient permissions?

Anyways, this token will always work :)

Reply:
Ah thanks, haven't tried it to be honest! I might do that, probably better to do with lower permissions.

Reply:
Hmm somehow the token seems to store nothing when I use it, getting this assertion error:
`assert os.environ.get("GITHUB_AUTH_TOKEN", "") != ""`

Don't know why, really ... any ideas?

Reply:
The nf_core_bot stuff is only available in the Github Actions, since it's a secret.
I didn't look into the failing Action, but this might be the issue.

Reply:
Sure, but we check in the code whether the environment variable `GITHUB_AUTH_TOKEN` is set, which should now contain the token. Hmm will do some more checks and just revert back to `GITHUB_TOKEN` if it doesn't work.